tests/helixperftasks.targets

```diff
@@ -33,6 +71,30 @@
 
   </Target>
 
+  <Target Name="RunTestsForProject">
+    <Message Text="This is a dummy target to appease DependsOnTargets for ArchiveTestBuild in publishtest.targets" Importance="Low" />
```

Comment:
I think it makes more sense to change things there versus make a fake target, but not blocking for me at least.